src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -329,7 +329,16 @@
     <Field Name="Index" Type="BoundExpression"/>
     <Field Name="Checked" Type="bool"/>
   </Node>
-  
+
+  <Node Name="BoundFunctionPointerInvocation" Base="BoundExpression">
+    <!-- Non-null type is required for this node kind -->
+    <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+    <Field Name="InvokedExpression" Type="BoundExpression"/>
+    <Field Name="FunctionPointer" Type="FunctionPointerTypeSymbol"/>
```

Comment:
><Field Name="FunctionPointer" Type="FunctionPointerTypeSymbol"/> [](start = 4, length = 64)

Wouldn't this be the type of ```InvokedExpression```?

Reply:
Yes, we could just use that.

---
In reply to: [382340027](https://github.com/dotnet/roslyn/pull/41794#discussion_r382340027) [](ancestors = 382340027)